src/style.css

```diff
@@ -0,0 +1,20 @@
+body {
+  margin: 0;
+}
+
+.general {
+  height: 100px;
+  width: 100%;
```

Comment:
it's a default value for `div`, no need to write it.

Reply:
replaced from ".general" to ".block"

Reply:
Ostap means `div` always have 100% width, you should delete this style rule